std/testing/bench_test.ts

```diff
@@ -338,6 +338,30 @@ test({
   },
 });
 
+test({
+  name: "async progressCallback",
+  fn: async function (): Promise<void> {
+    clearBenchmarks();
+    dummyBench("single");
+
+    const asyncCallbacks = [];
+
+    await runBenchmarks(
+      { silent: true },
+      (progress) => {
+        return new Promise(resolve => {
+          setTimeout(() => {
+            asyncCallbacks.push(progress);
+            resolve();
+          }, 10);
```

Comment:
Maybe use window.queueMicrotask?

Reply:
Ok, will change it.

Reply:
One question, is simply `queueMicrotask()` okay, or should i add the `window.` as well? Havent used `queueMicrotask` before, tests check out without window.

Reply:
leaving out the window is preferable - just specified that to be clear that it's global

Reply:
Understood, thanks!